docs/docs/admin/teamforcapella/migration.md

```diff
@@ -36,8 +41,19 @@
        on the old TeamForCapella server.
        ![Disable backup option](../../user/tools/capella/migration/backup-migration.png)
 
-1.  Export the model to the new TeamForCapella server via the Export wizzard
+1.  Export the model to the new TeamForCapella server via the Export wizard
     from TeamForCapella:
     [Export a model to TeamForCapella](../../user/tools/capella/teamforcapella/export/index.md)
 1.  Recreate the backup pipeline:
-    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md)
+    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md).
+    Start a pipeline run and verify that the changes are included in the Git
+    repository.
+1.  Use the [capella-diff-tools](https://github.com/DSD-DBS/capella-diff-tools)
+    and compare the commit after the migration with the commit before the
+    migration. Verify that the changes are correct.
```

Comment:
I see two missing steps here:

1. Remove the repository on the old T4C server
2. Remove the connected Capella projects from all user workspaces, when they point to the olds (not existing) T4C repo.

For the latter I suggest that a dedicated tooling will be developed.

Background:
We had a user who opened a Capella session and during startup Capella restored the state of his last workspace session. That came with opening the last model session state for the foo_bar.team project, that pointed to the old T4C repo (which was not existing anymore). That invalid situation led to a Capella freeze which was not easy to exit.

Reply:
Automatic tooling goes into the direction of workspace management: https://github.com/DSD-DBS/capella-collab-manager/issues/998. It needs some infrastructure to access workspaces.